Templates/Per Tenant Extension/.github/workflows/PullRequestHandler.yaml

```diff
@@ -3,6 +3,24 @@ name: 'Pull Request Build'
 on:
   pull_request_target:
     branches: [ 'main' ]
+    types:
+      - assigned
```

Comment:
From https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

> By default, a workflow only runs when a pull_request_target event's activity type is opened, synchronize, or reopened

This list contains the events that would trigger a new Pull Request Build run. Why would you want a new build triggered when the PR is labeled, or assigned?

Reply:
Oh, sorry, my mistake. I assumed the default array of types would be all of them, not just those. I do want to include `ready_for_review`, because that's when the PR exits draft mode.

Reply:
Assuming drafts don't trigger the PR handler, there wouldn't be a successful PR build, hence the need to trigger as soon as the PR leaves draft state. I agree that you would trigger a superfluous build by setting a ready PR to draft and back, but that's an edge case.

Reply:
From the other comment, how would not trigger PR Builds on draft PRs?

Reply:
I've now updated all jobs of the workflow with the draft check. If there's some job that shouldn't be skipped due to a particular reason, let me know and we can think of a solution.